sdks/python/setup.py

```diff
@@ -85,7 +85,6 @@ def get_version():
         'apache_beam/utils/counters.py',
         'apache_beam/utils/windowed_value.py',
     ]),
-    setup_requires=['nose>=1.0'],
```

Comment:
Excluding the nose folder in the pom.xml from RAT fixed this for me: https://github.com/aaltay/incubator-beam/commit/752d689335e606e3ef1ce40e07e8f89b59eafed3